recipe/meta.yaml

```diff
@@ -50,12 +50,16 @@ requirements:
 
 test:
   commands:
+    # Test plotting basemap with coastlines
     - gmt defaults -Vd
     - gmt psbasemap -R10/70/-3/8 -JX4i/3i -Ba -B+glightred+t"TEST" -P -Vd > test0.ps
     - gmt pscoast -R0/10/0/10 -JM6i -Ba -Ggray -ENG+p1p,blue -P -Vd > test1.ps
     - gmt begin -Vd
     - gmt coast -R0/10/0/10 -JM6i -Ba -Ggray -ENG+p1p,blue -Vd
     - gmt end
+    # Test OpenMP multi-core with -x2
+    - gmt-config --all
```

Comment:
Running `gmt-config --all` doesn't work on Windows, got an error like `'gmt-config' is not recognized as an internal or external command, operable program or batch file.` at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=758392&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f&l=5145. Is there an alternative way to check if OpenMP is enabled on Windows?

Expected results from Linux at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=758392&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=4214:

```
This GMT 6.4.0 has been built with the following features:

  --cc          -> /home/conda/feedstock_root/build_artifacts/gmt_1691472560800/_build_env/bin/x86_64-conda-linux-gnu-cc
  --cflags      -> -I$PREFIX/include/gmt
  --libs        -> -L$PREFIX/lib -lgmt

  --bits        -> 64
  --datadir     -> Not set
  --dataserver  -> http://oceania.generic-mapping-tools.org/
  --dcw         -> $PREFIX/share/dcw-gmt
  --dep-libs    -> $PREFIX/lib/libnetcdf.so $PREFIX/lib/libcurl.so $PREFIX/lib/libgdal.so $PREFIX/lib/libgeos_c.so $PREFIX/lib/libpcre.so $PREFIX/lib/libfftw3f.so $PREFIX/lib/libfftw3f_threads.so $PREFIX/lib/liblapack.so $PREFIX/lib/libblas.so $PREFIX/lib/libz.so
  --gshhg       -> $PREFIX/share/gshhg-gmt
  --has-fftw    -> yes
  --has-pcre    -> yes
  --has-lapack  -> yes
  --has-openmp  -> yes

  --prefix      -> $PREFIX
  --includedir  -> $PREFIX/include/gmt
  --libdir      -> $PREFIX/lib
  --plugindir   -> $PREFIX/lib/gmt/plugins
  --version     -> 6.4.0
```

Reply:
> Running [`gmt-config --all`](https://docs.generic-mapping-tools.org/6.4/gmt-config.html) doesn't work on Windows, got an error like `'gmt-config' is not recognized as an internal or external command, operable program or batch file.` at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=758392&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f&l=5145. Is there an alternative way to check if OpenMP is enabled on Windows?

Just run `gmt`. If compiled with OpenMP support, you will see something like:
```
GMT - The Generic Mapping Tools, Version 6.4.0 [64-bit] [MP] [80 cores]
```
otherwise, you'll see
```
GMT - The Generic Mapping Tools, Version 6.4.0 [64-bit] [80 cores]
```